docs/guide/server.md

```diff
@@ -0,0 +1,122 @@
+# Using Dolos Server
+
+We provide a free online service to run Dolos directly in our browser.
+You can use this service at [dolos.ugent.be/server](https://dolos.ugent.be/server).
+
+If you want to follow along, you can download a [ZIP with sample files](/simple-dataset.zip) which you can analyze on our server.
+
+## Upload and analyze files
+
+![Screenshot of the upload form](/images/screenshots/dolos-upload.png)
+
+To start a plagiarism analysis, upload a ZIP-file containing the files you want to analyze.
+You can specify a name for your analysis, which will be used to identify it in the report history.
+
+Additionally, you can select the programming language of the files.
+Dolos will automatically detect the language based on the file extension, but you can override this if needed.
+
+Click "Analyze" to upload your files and start the analysis.
+
+
+### Upload format
+
+Currently, we only support uploading a single ZIP-file containing all the files you want to analyze with an required `info.csv` file.
+This is the format in which [Dodona](./dodona.md) exports submissions.
+
+A column **filename** is required and should contain the path of the file relative to the root of the ZIP-file.
+
+The following columns are optional, but will be used to improve the analysis:
+- **labels** single label, submissions with the same label will have the same colour in the report.
+- **created_at** with the timestamp of the submission, this will be used to construct a timeline of the submissions.
+
+Additional columns will be ignored.
+
+#### Example
+
+An example of the `info.csv` for our sample submissions:
+
+::: code-group
+
+```csv [Only required column]
+filename
+sample.js
+copy_of_sample.js
+copied_function.js
+another_copied_function.js
+```
+
+```csv [With optional columns]
+filename,labels,created_at
+sample.js,original,2019-07-23 17:12:33 +0200
+copy_of_sample.js,copy,2019-07-25 11:02:57 +0200
+copied_function.js,copy,2019-07-25 14:43:20 +0200
+another_copied_function.js,copy,2019-07-27 19:22:39 +0200
```

Comment:
Using "copy" and "original" as a label example might be confusing to users. They might think that you somehow have to manually label the plagiarism before running dolos as some kind of training.